apps/web/src/app/vault/individual-vault/vault-filter/services/vault-filter.service.ts

```diff
@@ -47,12 +49,16 @@ const NestingDelimiter = "/";
 
 @Injectable()
 export class VaultFilterService implements VaultFilterServiceAbstraction {
-  private activeUserId$ = this.accountService.activeAccount$.pipe(map((a) => a?.id));
+  protected activeUserId$ = getUserId(this.accountService.activeAccount$);
 
   memberOrganizations$ = this.activeUserId$.pipe(
     switchMap((id) => this.organizationService.memberOrganizations$(id)),
   );
 
+  collapsedFilterNodes$ = this.activeUserId$.pipe(
+    switchMap((id) => this.collapsedGroupingsState(id).state$.pipe(map((c) => new Set(c)))),
+  );
+
```

Comment:
⛏️ we should avoid nesting pipes:
```suggestion
 collapsedFilterNodes$ = this.activeUserId$.pipe(
    switchMap((id) => this.collapsedGroupingsState(id).state$),
    map((state) => new Set(state))
  );
```

Reply:
Thank you!